### PR TITLE
Compatible single client rm useless interface

### PR DESCRIPTION
**Pull request by doombeaker**
非 `python/nn` 目录下，但是没有移除的 API 及其原因解释：
已经和 master 失联，看替代的：https://github.com/Oneflow-Inc/oneflow/pull/5556

| 保留的接口                                                   | 原因                                                         |
| ------------------------------------------------------------ | ------------------------------------------------------------ |
| ./python/framework/distribute.py:@oneflow_export("distributed.get_local_rank") | 这个文件新老接口混杂，保留的都是显示最新修改的，这个文件可能有误删 |
| ./python/framework/distribute.py:@oneflow_export("distributed.is_multi_client") |                                                              |
| ./python/framework/distribute.py:@oneflow_export("sbp.split") |                                                              |
| ./python/framework/distribute.py:@oneflow_export_value("sbp.broadcast") |                                                              |
| ./python/framework/distribute.py:@oneflow_export_value("sbp.partial_sum") |                                                              |
|                                                              |                                                              |
| ./python/framework/check_point_v2.py:@oneflow_export("get_all_variables") | 加载保存模型的新接口（利用eager），感觉与 multiclient 有关   |
| ./python/framework/check_point_v2.py:@oneflow_export("checkpoint.get") |                                                              |
| ./python/framework/check_point_v2.py:@oneflow_export("load") |                                                              |
| ./python/framework/check_point_v2.py:@oneflow_export("checkpoint.save") |                                                              |
| ./python/framework/check_point_v2.py:@oneflow_export("save") |                                                              |
| ./python/framework/check_point_v2.py:@oneflow_export("load_variables") |                                                              |
|                                                              |                                                              |
| ./python/framework/sysconfig.py:@oneflow_export("sysconfig.get_include") | 与系统编译有关的，和 lazy 无关，就保留了                     |
| ./python/framework/sysconfig.py:@oneflow_export("sysconfig.get_lib") |                                                              |
| ./python/framework/sysconfig.py:@oneflow_export("sysconfig.get_compile_flags") |                                                              |
| ./python/framework/sysconfig.py:@oneflow_export("sysconfig.get_link_flags") |                                                              |
| ./python/framework/sysconfig.py:@oneflow_export("sysconfig.with_cuda") |                                                              |
| ./python/framework/sysconfig.py:@oneflow_export("sysconfig.with_xla") |                                                              |
| ./python/framework/sysconfig.py:@oneflow_export("sysconfig.has_rpc_backend_grpc") |                                                              |
| ./python/framework/sysconfig.py:@oneflow_export("sysconfig.has_rpc_backend_local") |                                                              |
|                                                              |                                                              |
| ./python/framework/tensor.py:@oneflow_export("tensor")       | 新接口无疑                                                   |
| ./python/framework/tensor.py:@oneflow_export("Tensor")       |                                                              |
|                                                              |                                                              |
| ./python/framework/profiler.py:@oneflow_export("profiler.range_push") | 与性能测试有关，也可用于新接口测试，所以保留了               |
| ./python/framework/profiler.py:@oneflow_export("profiler.range_pop") |                                                              |
| ./python/framework/env_util.py:@oneflow_export("enable_eager_execution") |                                                              |
|                                                              |                                                              |
| ./python/framework/model.py:@oneflow_export("model.DataModule") | 看代码貌似与动静转化有关，就保留了                           |
| ./python/framework/model.py:@oneflow_export("model.NumpyDataModule") |                                                              |
| ./python/framework/model.py:@oneflow_export("model.TrainingConfig") |                                                              |
| ./python/framework/model.py:@oneflow_export("model.ValidationConfig") |                                                              |
| ./python/framework/model.py:@oneflow_export("model.CheckpointConfig") |                                                              |
| ./python/framework/model.py:@oneflow_export("model.Callback") |                                                              |
| ./python/framework/model.py:@oneflow_export("Model", "model.Model") |                                                              |
|                                                              |                                                              |
| ./python/framework/unittest.py:@oneflow_export("unittest.register_test_cases") | 单元测试有关，统统保留了                                     |
| ./python/framework/unittest.py:@oneflow_export("unittest.num_nodes_required") |                                                              |
| ./python/framework/unittest.py:@oneflow_export("unittest.env.eager_execution_enabled") |                                                              |
| ./python/framework/unittest.py:@oneflow_export("unittest.env.typing_check_enabled") |                                                              |
| ./python/framework/unittest.py:@oneflow_export("unittest.env.node_list") |                                                              |
| ./python/framework/unittest.py:@oneflow_export("unittest.env.has_node_list") |                                                              |
| ./python/framework/unittest.py:@oneflow_export("unittest.env.node_size") |                                                              |
| ./python/framework/unittest.py:@oneflow_export("unittest.env.has_world_size") |                                                              |
| ./python/framework/unittest.py:@oneflow_export("unittest.env.world_size") |                                                              |
| ./python/framework/unittest.py:@oneflow_export("unittest.env.device_num") |                                                              |
| ./python/framework/unittest.py:@oneflow_export("unittest.TestCase") |                                                              |
| ./python/framework/unittest.py:@oneflow_export("unittest.skip_unless_1n1d") |                                                              |
| ./python/framework/unittest.py:@oneflow_export("unittest.skip_unless_1n2d") |                                                              |
| ./python/framework/unittest.py:@oneflow_export("unittest.skip_unless_1n4d") |                                                              |
| ./python/framework/unittest.py:@oneflow_export("unittest.skip_unless_2n1d") |                                                              |
| ./python/framework/unittest.py:@oneflow_export("unittest.skip_unless_2n2d") |                                                              |
| ./python/framework/unittest.py:@oneflow_export("unittest.skip_unless_2n4d") |                                                              |
|                                                              |                                                              |
| ./python/framework/dtype.py:@oneflow_export("dtypes")        | 数据类型有关，保留了                                         |
| ./python/framework/dtype.py:@oneflow_export("convert_oneflow_dtype_to_numpy_dtype") |                                                              |
|                                                              |                                                              |
| ./python/framework/generator.py:@oneflow_export("Generator") | 最近后江、陈博文新增的与随机数有关的方法，保留了             |
| ./python/framework/generator.py:@oneflow_export("default_generator") |                                                              |
| ./python/framework/generator.py:@oneflow_export("manual_seed") |                                                              |
|                                                              |                                                              |
| ./python/autograd/autograd.py:@oneflow_export("autograd.grad") | 新接口无疑                                                   |
| ./python/autograd/autograd.py:@oneflow_export("autograd.backward") |                                                              |
|                                                              |                                                              |
| ./python/experimental/load_mnist.py:@oneflow_export("data.load_mnist") | 加载 numpy 格式的 mnist 数据集，保留了                       |
|                                                              |                                                              |
| ./python/oneflow_export.py:@oneflow_export("is_deprecated")  | 判断接口是否过期，感觉可去可不去，暂时保留了                 |

### CODE REVIEW

File: oneflow/python/contrib/tensorrt/tensorrt_api.py [hunk 1]
```diff
@@ -20,15 +20,13 @@
 import oneflow._oneflow_internal
 
 
-@oneflow_export("tensorrt.write_int8_calibration")
```
Comment (clackhan) [inline oneflow/python/contrib/tensorrt/tensorrt_api.py:23]: 这里涉及到tensorrt的先不要删

File: oneflow/python/deprecated/init_cluster_env.py [hunk 2]
```diff
@@ -29,7 +29,6 @@
 import subprocess
 
 
-@oneflow_export("deprecated.delete_worker_by_bootstrap")
```
Comment (clackhan) [inline oneflow/python/deprecated/init_cluster_env.py:32]: 这个文件里的接口主要是ci和unittest用到，暂时也不要删

File: oneflow/python/framework/distribute.py [hunk 3]
```diff
@@ -186,13 +170,11 @@ def assert_is_valid_distribute(
     ), """not a valid distribute policy.
            expected: 1) oneflow.distribute.split(axis); 2) oneflow.distribute.broadcast(); 3) oneflow.distribute.auto()"""
 
-
 @oneflow_export("distributed.get_local_rank")
 def get_local_rank():
     return oneflow._oneflow_internal.GetLocalRank()
 
 
-@oneflow_export("distributed.get_rank")
```
Comment (clackhan) [inline oneflow/python/framework/distribute.py:195]: 保留

File: oneflow/python/framework/distribute.py [hunk 4]
```diff
@@ -203,7 +185,6 @@ def get_rank():
     return oneflow._oneflow_internal.GetRank()
 
 
-@oneflow_export("distributed.get_world_size")
```
Comment (clackhan) [inline oneflow/python/framework/distribute.py:206]: 保留

File: oneflow/python/framework/env_util.py [hunk 5]
```diff
@@ -33,7 +33,6 @@
 import traceback
 
 
-@oneflow_export("env.all_device_placement")
```
Comment (clackhan) [inline oneflow/python/framework/env_util.py:36]: 保留

File: oneflow/python/framework/env_util.py [hunk 6]
```diff
@@ -293,7 +281,6 @@ def _MakeMachine(machines):
 
 
 # only used by CI
-@oneflow_export("env.init_bootstrap_confs")
```
Comment (clackhan) [inline oneflow/python/framework/env_util.py:296]: 保留

File: oneflow/python/framework/session_util.py [hunk 7]
```diff
@@ -475,7 +473,6 @@ def api_eager_execution_enabled() -> bool:
     return oneflow._oneflow_internal.EagerExecutionEnabled()
 
 
-@oneflow_export("clear_default_session")
```
Comment (clackhan) [inline oneflow/python/framework/session_util.py:478]: 保留

File: oneflow/python/ops/builtin_ops.py [hunk 8]
```diff
@@ -23,7 +23,6 @@
 from oneflow.python.oneflow_export import oneflow_export
 
 
-@oneflow_export("builtin_op")
```
Comment (clackhan) [inline oneflow/python/ops/builtin_ops.py:26]: 保留

File: oneflow/python/serving/inference_session.py [hunk 9]
```diff
@@ -122,20 +122,17 @@ def _inferface_blob_conf_proto_to_cfg(
     mut_inferface_blob_conf_cfg.set_is_dynamic(inferface_blob_conf_proto.is_dynamic)
 
 
-@oneflow_export("serving.ModelVersionPolicy")
```
Comment (clackhan) [inline oneflow/python/serving/inference_session.py:125]: serving名字空间下的先保留

File: oneflow/python/serving/saved_model_builder.py [hunk 10]
```diff
@@ -30,7 +30,6 @@
 from oneflow.python.oneflow_export import oneflow_export
 
 
-@oneflow_export("saved_model.ModelBuilder")
```
Comment (clackhan) [inline oneflow/python/serving/saved_model_builder.py:33]: serving名字空间下的先保留

File: oneflow/python/contrib/tensorrt/tensorrt_api.py (hunk 1 above)
Comment (doombeaker) [reply oneflow/python/contrib/tensorrt/tensorrt_api.py:23]: 已修改

File: oneflow/python/deprecated/init_cluster_env.py (hunk 2 above)
Comment (doombeaker) [reply oneflow/python/deprecated/init_cluster_env.py:32]: 已经恢复

File: oneflow/python/framework/distribute.py (hunk 3 above)
Comment (doombeaker) [reply oneflow/python/framework/distribute.py:195]: 已恢复

File: oneflow/python/framework/distribute.py (hunk 4 above)
Comment (doombeaker) [reply oneflow/python/framework/distribute.py:206]: 已恢复

File: oneflow/python/framework/env_util.py (hunk 5 above)
Comment (doombeaker) [reply oneflow/python/framework/env_util.py:36]: 已恢复

File: oneflow/python/framework/env_util.py (hunk 6 above)
Comment (doombeaker) [reply oneflow/python/framework/env_util.py:296]: 已恢复

File: oneflow/python/framework/session_util.py (hunk 7 above)
Comment (doombeaker) [reply oneflow/python/framework/session_util.py:478]: 已恢复

File: oneflow/python/serving/saved_model_builder.py (hunk 10 above)
Comment (doombeaker) [reply oneflow/python/serving/saved_model_builder.py:33]: 已恢复

File: oneflow/python/serving/inference_session.py (hunk 9 above)
Comment (doombeaker) [reply oneflow/python/serving/inference_session.py:125]: 已恢复

**Pull request closed without merging**